### DISCUSSION TITLE
Will there be update for Ventura in future for Elitebook 2570p ?

### DISCUSSION THREAD

**Discussion by kanthi** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#150] [General]
Just wanted to know if this the end for 2570p

**Comment by chris1111** [comment]
No the hope exist! [OCLP](https://github.com/dortania/OpenCore-Legacy-Patcher/) has already made a lot of progress, I will look into how I should apply the patch for HP hackintosh Laptops. but it's going to be a long process, be patient.

**Reply by kanthi** [reply]
Any updates on this pls...

**Comment by chris1111** [comment]
Its updated

**Reply by kanthi** [reply]
Thank You and appreciate your efforts
